tests/test_errors.py

```diff
@@ -517,6 +521,137 @@ def test_all_errors():
         pytest.fail('core_schema.ErrorType needs to be updated')
 
 
+def test_validation_error_cause_usage():
+    if sys.version_info < (3, 11):
+        from exceptiongroup import BaseExceptionGroup
+    else:
+        from builtins import BaseExceptionGroup
+
+    enabled_config: CoreConfig = {'validation_error_cause': True}
+
+    def check_grouped_exception(exc: BaseException) -> BaseException:
```

Comment:
better to have two tests and add add a `pytest.mark.skipif` to both, so only one runs.

Reply:
I've separated out the contents tests as you suggested here :)